projects/demo/src/modules/components/dialog/dialog.component.ts

```diff
@@ -89,6 +90,11 @@ export class ExampleTuiDialogComponent {
         HTML: example6Html,
     };
 
+    readonly example7: TuiDocExample = {
+        TypeScript: import('!!raw-loader!./examples/7/index.ts'),
+        HTML: import('!!raw-loader!./examples/7/index.html'),
```

Comment:
add `SearchDialogExampleComponent`-files (otherwise stackblitz will not work)

Reply:
fixed